Add yield, order by score to fts doc (#426)
kuzudb/kuzu-docs: src/content/docs/extensions/full-text-search.md

to an optional parameter in the example below.
2. Users can only build full text search indexes on node tables.
3. Once a full-text search index is created, the stopword list becomes immutable. All queries against the index will permanently reference the original stopword list used during its creation. To update the stopword configuration, you must rebuild the index with the revised stopword list.

> 4. `CREATE_FTS_INDEX` can only be executed in a query which doesn't have other statements.

Reviewer: ```suggestion
4. `CREATE_FTS_INDEX` cannot be executed in a multi-statement query block that also includes other statements.
```

> E.g. 

Reviewer: ```suggestion
For example:
```

- `INDEX_NAME`: The name of the FTS index to query. 
- `QUERY`: The query string that contains the keywords to search.

> Users can use `YIELD` to rename the result columns. More details on `YIELD` can be found [here](cypher/query-clauses/call/#using-yield).

Reviewer: ```suggestion
You can use `YIELD` to rename the result columns. More details on `YIELD` can be found [here](cypher/query-clauses/call/#using-yield).
```
